frontend/src/containers/RecipePage/Recipe.js

```diff
@@ -0,0 +1,151 @@
+      );
+    }
+    return undefined; // TODO: This is probably bad. What should this return for invalid difficulty?
+  };
+
+    return (
+      <div className="overall-container">
+        <div className="row">
+          <div className="header">
+            <Header />
+          </div>
+        </div>
+        <div>
+            <button type="button" className="user-edit-button">
+              <img 
+                src="https://img.icons8.com/windows/32/ffa616/edit.png"
+                alt="edit"
+              />
+
+            </button>
+            <button type="button" className="user-edit-button">
+              <img
+                src="https://img.icons8.com/windows/32/ffa616/cancel.png"
+                alt="delete"
+              />
+            </button>
+        </div>
+        <img src={recipephoto} alt="recipephoto" className="recipe-photo" />
+          
+          <div className="specs-bars">
+          <div className="row">
+         
+          <div id="diff-spec-container1" className="spec-element col-sm-4 text-center">
+              <span className="card-meta"> {mealDifficulty()}
+               {difficulty}</span>
+               </div>
+
+             <div id="diff-spec-container2" className="spec-element col-sm-4 text-center">
+             <span className="Bluez">
+              <i className="fas fa-clock" />
+                 { duration }{' '}MIN</span>
+               </div>
+
+
+          <div id="diff-spec-container3" className="spec-element col-sm-4 text-center">
+           <i className="fas fa-utensil-spoon" />
+              <span className="Bluez">{ servings }</span>
+            </div>
+
+            </div>
+           </div>
+
+        <div className="tags-bar">
+          <TagsBar />
+        </div>
+        <div className="row" id="blurb-title">
+          <h1 id="recipe-title" className="col-sm-6 offset-sm-3">
+             {title}
+          </h1>
+          <p id="recipe-descrip" className="col-sm-8 offset-sm-2">
+            {
+              snippet
+            }
+          </p>
+        </div>
+        <div id="hr" />
+        <div className="separator col-sm-10 offset-sm-1" />
+
+    
+
+        <div id="ingredient-box">
+          <div id="ingredient-list">
+            <h1 id="ingredient-title">Ingredients</h1>
+            <ul id="ingredient-ul">
+              {ingredient.map(ingred => (
+                <li>
+                  <div id="single-ingredient">{ingred}</div>
+                </li>
+              ))}
+            </ul>
+          </div>
+        </div>
+
+        <div id="hr" />
+        <div className="separator col-sm-10 offset-sm-1" />
+        <div id="direction-container" className="col-sm-10 offset-sm-1">
+          <h1 id="direction-title">Directions</h1>
+          <ul className="ListItems">
+            {directions.map((direction, index) => (
+              <div className="single-direction">
+                <h1 className="direction-index">{index + 1}</h1>
```

Comment:
What do you think about using `ol` for this in place and leveraging some of the ideas in https://css-tricks.com/custom-list-number-styling/ to manage the styling instead of `h1`? Particularly since this isn't, semantically speaking, a header.

Either way, I think the index should be smaller than the header, so probably want to use something that's not h1.